File: ClancySafeAI.Core/Models/Training.cs
namespace ClancySafeAI.Core.Models
{
    public class Training
    {
        public string Id { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public List<TrainingSlide> Slides { get; set; }
        public TrainingStatus Status { get; set; }
        public DateTime? CompletedAt { get; set; }
    }

    public enum TrainingStatus
    {
        NotStarted,
        InProgress,
        Completed
    }
} 

File: ClancySafeAI.Client/clancysafeai.client.client/src/types/training.ts
export interface TrainingSlide {
  id: number
  title: string
  content: string
  imageUrl?: string
  audioUrl?: string
  duration: number
}

export interface Training {
  id: string
  title: string
  description: string
  slides: TrainingSlide[]
  status: 'not_started' | 'in_progress' | 'completed'
  completedAt?: string
} 